docs/3.0.0-beta.x/getting-started/introduction.md

```diff
@@ -1,20 +1,18 @@
-# Introduction 
+# Introduction
 
 Welcome to the open source [headless CMS](https://strapi.io) developers love.
 
 ## Strapi Documentation
 
 ### 👋 Welcome onboard!
 
-Users love Strapi because it is open source, MIT licensed, fully customizable and based on Node.js. Strapi lets you manage your content and distribute it anywhere. Strapi allows you to securely and privately serve your database of choice from your hosting and server of choice. 
+Users love Strapi because it is open source, MIT licensed, fully customizable and based on Node.js. Strapi lets you manage your content and distribute it anywhere. Strapi allows you to securely and privately serve your database of choice from your hosting and server of choice.
```

Comment:
Strapi allows you to securely and privately serve your preferred database from your hosting and server of choice.

So we don't repeat of choice twice.